src/pke/examples/simple-ckks-bootstrapping.cpp

```diff
@@ -0,0 +1,202 @@
+//    list of conditions and the following disclaimer.
+//
+// 2. Redistributions in binary form must reproduce the above copyright notice,
+//    this list of conditions and the following disclaimer in the documentation
+//    and/or other materials provided with the distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+// DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+// OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//==================================================================================
+
+/*
+
+Example for CKKS bootstrapping
+
+*/
+
+#define PROFILE
+
+#include "openfhe.h"
+#include "scheme/ckksrns/cryptocontext-ckksrns.h"
+#include "gen-cryptocontext.h"
+
+using namespace std;
+using namespace lbcrypto;
+
+int main(int argc, char* argv[]) {
+    // Step 1: Set CryptoContext
+    CCParams<CryptoContextCKKSRNS> parameters;
+
+    // A. Specify main parameters
+    /*  A1) Secret key distribution
+    * The secret key distribution for CKKS should either be SPARSE_TERNARY or UNIFORM_TERNARY.
+    * The SPARSE_TERNARY distribution was used in the original CKKS paper,
+    * but in this example, we use UNIFORM_TERNARY because this is included in the homomorphic
+    * encryption standard.
+    */
+    SecretKeyDist secretKeyDist = UNIFORM_TERNARY;
+    parameters.SetSecretKeyDist(secretKeyDist);
+
+    /*  A2) Desired security level based on FHE standards.
+    * In this example, we use the "NotSet" option, so the example can run more quickly with
+    * a smaller ring dimension. Note that this should be used only in
+    * non-production environments, or by experts who understand the security
+    * implications of their choices. In production-like environments, we recommend using
+    * HEStd_128_classic, HEStd_192_classic, or HEStd_256_classic for 128-bit, 192-bit,
+    * or 256-bit security, respectively. If you choose one of these as your security level,
+    * you do not need to set the ring dimension.
+    */
+    parameters.SetSecurityLevel(HEStd_NotSet);
+    parameters.SetRingDim(1 << 12);
+
+    /*  A3) Key switching parameters.
+    * By default, we use HYBRID key switching with a digit size of 3.
+    * Choosing a larger digit size can reduce complexity, but the size of keys will increase.
+    */
+    parameters.SetNumLargeDigits(3);
```

Comment:
Add a note that commenting out these two lines will not change anything as they are used by default

Reply:
Added BootstrapExampleClean() without these lines.